.tool-versions

```diff
@@ -0,0 +1,2 @@
+ruby 3.3.4
```

Comment:
do you really want to change from .ruby-version to .tool-verisons?

Reply:
yeah i think so. most tools are supporting it now:
<img width="781" alt="image" src="https://github.com/user-attachments/assets/980ea9c3-f689-4323-a7f0-b82c7f4637c2">
https://mise.jdx.dev/configuration.html#tool-versions

it makes it easier to change versions for everything and just in case we forget to update the .ruby-version